exchanges/binance/binance_wrapper.go

```diff
@@ -598,10 +611,68 @@ func (b *Binance) CancelAllOrders(_ *order.Cancel) (order.CancelAllResponse, err
 	return cancelAllOrdersResponse, nil
 }
 
-// GetOrderInfo returns information on a current open order
-func (b *Binance) GetOrderInfo(orderID string) (order.Detail, error) {
-	var orderDetail order.Detail
-	return orderDetail, common.ErrNotYetImplemented
+// GetOrderInfo returns order information based on order ID
+func (b *Binance) GetOrderInfo(orderID string, pair currency.Pair, assetType asset.Item) (o order.Detail, err error) {
+	if assetType == "" {
+		assetType = asset.Spot
+	}
+
+	formattedPair, err := b.FormatExchangeCurrency(pair, assetType)
+	if err != nil {
+		return
+	}
+
+	orderIDInt64, err := convert.Int64FromString(orderID)
+	if err != nil {
+		return
+	}
+
+	resp, err := b.QueryOrder(formattedPair.String(), "", orderIDInt64)
+	if err != nil {
+		return
+	}
+
+	orderSide := order.Side(resp.Side)
+	orderDate, err := convert.TimeFromUnixTimestampFloat(resp.Time)
+	if err != nil {
+		return
+	}
+
+	orderCloseDate, err := convert.TimeFromUnixTimestampFloat(float64(resp.UpdateTime))
+	if err != nil {
+		return
+	}
+
+	orderPair, err := currency.NewPairFromString(resp.Symbol)
```

Comment:
This is formatting it wrong for THETA-USDT:
```
go run ./... getorder binance "153898513" THETA-USDT
{
 "exchange": "Binance",
 "id": "153898513",
 // "base_currency": "THE",
 // "quote_currency": "TAUSDT",
 "asset_type": "spot",
 "order_side": "BUY",
 "order_type": "LIMIT",
 "creation_time": 1603237752,
 "update_time": 1603237752,
 "status": "NEW",
 "price": 0.2,
 "amount": 2200
}
```

Maybe just use the formattedPair in order.Detail

Reply:
Right, thanks!